docs/add-ons/tutorial.md

```diff
@@ -117,12 +117,12 @@ RUN apk add --no-cache python3
 WORKDIR /data
 ```
 
-Add "ports" to `config.json`. This will make TCP on port 8000 inside the container available on the host on port 8000.
+Add "ports" to `config.json`. This will make TCP on port 8000 inside the container available on the host on port 8000. Please note that the version must be incremented as well. This way the supervisor will know that a new version of the add-on is available. 
```

Comment:
That is not needed, reloading the add-on store, however, is.

Reply:
Much faster than expected, hey @frenck ! 

So maybe I see it wrong, but however I am changing the config.json the supervisor will not see an available update even after the reload is done unless I am changing the version as well. 
Am I not noticing something? 
Thanks! 
B

Reply:
I dunno. My dev version is always `dev`. I've never changed versions during development ever.